index.html

```diff
@@ -5,9 +5,96 @@
 
     <link href="styles.css" rel="stylesheet" />
 
-    <title>Rocket Academy</title>
+    <title>Qian's Portfolio</title>
   </head>
   <body>
-    <h1>Hello, world!</h1>
+    <section class="header-section">
+      <div class="header">
+        <p id="author-name">Yap Qianling</p>
+        <div class="header-navigation">
+          <a href="#home">Home</a>
+          <a href="#projects">Projects</a>
+          <a href="#skills">Skills</a>
+          <a href="#contact">Contact</a>
+        </div>
+        <div>
+          <button class="hire-button">Contact Me ></button>
+        </div>
+      </div>
+
+    </section>
+
+    <section id="home" class="home-section">
+        
+      <div class="about">
+        <p class="title">
+          Hey there, I'm Qianling! <br />An aspiring Software Engineer.
```

Comment:
Personally I am not the biggest fan of `<br />`. I would rather make another `<p>` and use margins to create the desired distance between the lines